storage/interface.go

```diff
@@ -26,6 +26,9 @@ type VolumeParams struct {
 	// as defined in a storage pool.
 	Options map[string]interface{}
 
+	// The provider type for this volume.
+	VolumeType ProviderType
```

Comment:
I think just call the field Provider. Or ProviderType. VolumeType sounds more like the EBS volume type.